### PR TITLE
bump version 5.0.3

### PR DESCRIPTION
**Pull request by Taym95**
@werediver @PieterAelse , can you please review this so we can release new version!

- [x] fix CI

### CODE REVIEW

File: example/pubspec.lock [hunk 1]
```diff
@@ -205,10 +205,10 @@ packages:
   flutter_reactive_ble:
     dependency: "direct main"
     description:
-      path: "../packages/flutter_reactive_ble"
-      relative: true
+      path: "/home/taym/workspace/opensource/flutter_reactive_ble/example/../packages/flutter_reactive_ble"
```
Comment (werediver) [inline example/pubspec.lock:208]: Nope, not approving this.

File: packages/flutter_reactive_ble/CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,8 @@
+## 5.0.3
+* Enable extended advertising on android. Fix #571
```
Comment (werediver) [inline packages/flutter_reactive_ble/CHANGELOG.md:2]: Empty line after the header, please.

File: packages/reactive_ble_mobile/CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,7 @@
+## 5.0.3
+* Enable extended advertising on android. Fix #571
```
Comment (werediver) [inline packages/reactive_ble_mobile/CHANGELOG.md:2]: If at all possible, an empty line after the header.

File: example/pubspec.lock (hunk 1 above)
Comment (Taym95) [reply example/pubspec.lock:208]: I dont know what is this!

File: packages/reactive_ble_platform_interface/CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,7 @@
+## 5.0.3
+* Enable extended advertising on android. Fix #571
```
Comment (werediver) [inline packages/reactive_ble_platform_interface/CHANGELOG.md:2]: Shall I suggest you a Markdown linter? :)

File: example/pubspec.lock (hunk 1 above)
Comment (werediver) [reply example/pubspec.lock:208]: A dependency override?

File: packages/reactive_ble_platform_interface/CHANGELOG.md (hunk 4 above)
Comment (Taym95) [reply packages/reactive_ble_platform_interface/CHANGELOG.md:3]: Yeah I think so :smile:

File: example/pubspec.lock [hunk 5]
```diff
@@ -407,16 +407,16 @@ packages:
...
   reactive_ble_platform_interface:
-    dependency: transitive
+    dependency: "direct overridden"
     description:
-      name: reactive_ble_platform_interface
-      url: "https://pub.dartlang.org"
-    source: hosted
-    version: "5.0.2"
+      path: "../packages/reactive_ble_platform_interface"
+      relative: true
```
Comment (werediver) [inline example/pubspec.lock:417]: I, personally, prefer not to commit dependency overrides (or straight path-dependencies). Unless you have an explicit desire to change what's in the repo from how it was to this, I'd suggest to not commit any overrides and not use straight path-dependencies.

File: packages/reactive_ble_platform_interface/CHANGELOG.md [hunk 6]
```diff
@@ -1,3 +1,8 @@
+## 5.0.3
+
+* Enable extended advertising on android. Fix #571
+* Add additional null checks to scanRecord. Fix #521
+* Restore support for Xcode 12
 ## 5.0.2
```
Comment (werediver) [inline packages/reactive_ble_platform_interface/CHANGELOG.md:6]: Omg, no empty line _before_ a header?.. 🙈

**Review by werediver — APPROVED** [review]

File: packages/reactive_ble_platform_interface/CHANGELOG.md (hunk 4 above)
Comment (werediver) [reply packages/reactive_ble_platform_interface/CHANGELOG.md:3]: I have this one installed (in VS Code): https://marketplace.visualstudio.com/items?itemName=DavidAnson.vscode-markdownlint

**Review by PieterAelse — APPROVED** [review]

**Comment by ened** [comment]
@Taym95 can you fix the CI?

**Comment by RtgrV** [comment]
> @Taym95 can you fix the CI?

It was resolved today, sorry for the delay.

**Pull request merged**